tests/development/authentication/test_401_errors.py

```diff
@@ -10,7 +10,20 @@
 @pytest.mark.parametrize("method", METHODS)
 def test_401_invalid_level(nhsd_apim_proxy_url, nhsd_apim_auth_headers, correlation_id, method):
     """
-    .. py:function:: Test 401 application has invalid level
+    .. py:function:: Scenario: An API consumer submitting a request with an insufficient \
+        level of authentication receives a 401 'Access Denied' response
+
+        | **Given** the API consumer provides an insufficient level of authentication

```

Comment:
should these all be partials seeing as the content is duplicated across all of the test suites?